services/twilioService.js

```diff
@@ -127,7 +127,13 @@ class TwilioService {
 
   async updateMessages(contact) {
     const channel = await this.chatClient.getChannelBySid(contact.channelSid);
-    const messages = await channel.getMessages();
+    const messages = await channel.getMessages(10);
```

Comment:
@bongiovimatthew this is set to `10` just for testing, you can change the number before merging